orgs/SovereignCloudStack/data.yaml

```
@@ -57,6 +57,12 @@ members:
   - name: ""
     login: janiskemper
     role: member
+  - name: ""
```

```suggestion
  - name:  Aniruddha Basak
```
Reasoning: The name is public on his github profile, so no need to hide it here.